src/test/java/WebDriverSeleniumTests/GoogleSearchForWikiChrome.java

```diff
@@ -0,0 +1,90 @@
+package WebDriverSeleniumTests;
```

Comment:
see my comments from Edge area - here is the same picture

Reply:
yep.